cypress/e2e/login/login.spec.ts

```diff
@@ -47,6 +47,30 @@ describe("Login", () => {
             // wait for the dialog to go away
             cy.get(".mx_ServerPickerDialog").should("not.exist");
 
+            cy.get(".mx_Spinner").should("not.exist");
+            cy.get(".mx_ServerPicker_server").should("have.text", homeserver.baseUrl);
+
+            cy.findByRole("button", { name: "Edit" }).click();
+
+            // select the default server again
```

Comment:
It'd be great if the comment could be clear that this is here as a regression test, otherwise it just looks like an unnecessary extra step in the context of the the test name